tests/model/test_model.py

```diff
@@ -656,33 +656,35 @@ def test_append_message(self, mocker, user_profile, response,
         # LOG REMAINS THE SAME IF UPDATE IS FALSE
         assert model.msg_list.log == log
 
-    @pytest.mark.parametrize('response, expected_message', [
+    @pytest.mark.parametrize('response, update_call_count, expected_message', [
         ({
             'subject': 'new subject',
-        }, {
+            'message_ids': [1],
+        }, 1, {
             'content': 'old content',
             'subject': 'new subject',
         }),
         ({
             'rendered_content': '<p>new content</p>',
-        }, {
+        }, 1, {
             'subject': 'old subject',
             'content': '<p>new content</p>',
         }),
         ({
             'rendered_content': '<p>new content</p>',
             'subject': 'new subject',
-        }, {
+            'message_ids': [1],
+        }, 2, {
             'subject': 'new subject',
             'content': '<p>new content</p>',
         }),
-        ({}, {
+        ({}, 0, {
             'subject': 'old subject',
             'content': 'old content',
         }),
     ])
```

Comment:
Two of these tests fail after this second commit; perhaps add some ids while you're doing this, and it will make it clearer?

Reply:
Done!